### PR TITLE
Add redux

### PR DESCRIPTION
**Pull request by varun13169**
Add redux

### CODE REVIEW

File: src/utils/ThemeSlice/themeSlice.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  currentTheme: ["light", "dark"].includes(localStorage.getItem("currentTheme"))
+    ? localStorage.getItem("currentTheme")
+    : "light",
```
Comment (iRohitGaur) [inline src/utils/ThemeSlice/themeSlice.js:6]: suggestion: this could be `currentTheme: localStorage.getItem("currentTheme") === "dark" ? "dark" : "light"`

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/utils/ThemeSlice/themeSlice.js (hunk 1 above)
Comment (varun13169) [reply src/utils/ThemeSlice/themeSlice.js:6]: My intent was to add more themes going forward, so it'll not be binary.

**Review by zoef-yt — APPROVED** [review]
Small pr just added redux. you are good to merge. good work.

**Pull request merged**